doc/api/http2.md

```diff
@@ -232,10 +232,12 @@ session.on('remoteSettings', (settings) => {
 added: v8.4.0
 -->
 
-The `'stream'` event is emitted when a new `Http2Stream` is created. When
-invoked, the handler function will receive a reference to the `Http2Stream`
-object, a [HTTP/2 Headers Object][], and numeric flags associated with the
-creation of the stream.
+* `stream` {Http2Stream} A reference to the stream
+* `headers` {HTTP/2 Headers Object} An object describing the headers
+* `flags` {number} The associated numeric flags
+* `rawHeaders` {Array} An array containing the raw header name value pairs
```

Comment:
Is this actually supposed to be public? @nodejs/http2

Reply:
I am not sure what is the convention here, but maybe `name value` -> `name-value` or `name/value` or `name:value` (with or without spaces, I am not sure as well)?

Reply:
Yes it's correct. `rawHeaders` is exposed as `request.rawHeaders`.

Reply:
@vsemozhetbyt I think it could be either depending on the key:value delimiter? Let me check what the delimiter is in this particular case.